src/OmniSharp.Roslyn/Utilities/ShadowCopyAnalyzerAssemblyLoader.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.ExternalAccess.OmniSharp.Analyzers;
+
+namespace OmniSharp.Roslyn.Utilities
+{
+    public static class ShadowCopyAnalyzerAssemblyLoader
+    {
+        public static readonly IAnalyzerAssemblyLoader Instance = OmnisharpAnalyzerAssemblyLoaderFactory.CreateShadowCopyAnalyzerAssemblyLoader();
```

Comment:
This is here because the Omnisharp.Host project doesn't have internals access to Microsoft.CodeAnalysis.ExternalAccess.Omnisharp; I don't know if that was an accidental oversight or intentional layering, but following it for now.

Reply:
I'm sure it just wasn't necessary for the initial round of reflection removal. This seems fine for now.